File: 0.C_CPP_JAVA Programs/JAVA/Program25.java
/*
4. Write Java program which accept number of rows and number of columns from user and display below pattern.
Input :  iRow = 4  iCol = 5
Output : 4 4 4 4 4
         3 3 3 3 3
         2 2 2 2 2
         1 1 1 1 1
 */
import java.lang.*;
import java.util.*;

class DemoString
{
    void Pattern(int No1,int No2)
    {
        for (int i=0;i<No1;No1--)
        {
            for(int j=0;j<No2;j++)
            {
                    System.out.print("\t" + No1);
            }
            System.out.print("\n");
        }
    }
}
class Demo
{
    public static void main(String arg[])
    {

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the Coloumn:");
        int No1=sobj.nextInt();

        System.out.println("Enter second row:");
        int No2=sobj.nextInt();

        DemoString dobj=new DemoString();

         dobj.Pattern(No1,No2);
    }
}

File: 1.Python/Program25.py
# 3.Write a program which accept N numbers from user and store it into List.
# Return Minimum number from that List.
# Input : Number of elements : 4
# Input Elements : 13 5 45 7
# Output : 5
def Min(arr):
    MinNo=arr[0]
    for i in range(len(arr)):
        if MinNo>arr[i]:
            MinNo=arr[i]
    return MinNo
def main():
    size=0
    arr=[]
    i=0
    print("Enter the number of element:")
    size=int(input())
    print("Enter the elements:")
    for i in range(size):
        arr.append(int(input()))
    ret=Min(arr);
    print("Minimum is:",ret)
if __name__=="__main__":
    main()